### ISSUE TITLE
test-aws-ubuntu1804-armv8-1 missing language dependency for test

### ISSUE THREAD

**Issue by M-Davies** [AdoptOpenJDK/openjdk-infrastructure#1268]
https://github.com/eclipse/openj9/issues/9042

Test `java/util/Locale/LocaleCategory.java` has been failing on this machine due to a system language issue.

See https://github.com/eclipse/openj9/issues/9042#issuecomment-611503587. It appears to be the LANG variable that is one factor that is causing this test to fail. 

However, it is likely not the only factor.`LANG` is set to `C.UTF-8` (the same `LANG` setting as `test-aws-ubuntu1804-armv8-1`) on `cent7-aarch64-1` and the test passes. The purpose of this issue is to determine if there is something missing in the dependencies of this machine that is causing this test to fail and, if not, discuss setting the `LANG` variable to `en_US.UTF-8` OR `en_GB.UTF-8` to resolve the issue.

**Comment by sxa** [comment]
@Haroon-Khel Can you take a look at this and see if there's anything obvious that might be missing on [test-aws-ubuntu1804-armv8-1](https://ci.adoptopenjdk.net/computer/test-aws-ubuntu1804-armv8-1) (`34.243.202.254`) - Hopefully that failure will be easy enough to reproduce when you're logged in.

**Comment by Haroon-Khel** [comment]
I was able to recreate the test error locally on test-aws-ubuntu1804-armv8-1. I cant find anything obvious thats missing on the machine. Even after running the Locales tasks, from the unix playbook, the test continues to fail

**Comment by Haroon-Khel** [comment]
@knn-k You're right. On test-aws-ubuntu1804-armv8-1, I have set the `LANG` variable to `en_US.UTF-8` in `/etc/default/locale` and the test passes. @M-Davies is it safe to say this issue can be closed?

**Comment by Haroon-Khel** [comment]
Odd, the test passes locally but still fails in the grinder with the original error
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2875/testReport/java_util_Locale_LocaleCategory/java/LocaleCategory/

**Comment by jdekonin** [comment]
A possible issue, looks like the machine was connected to jenkins on April 1st and this is an environment setting change.  I believe you need to disconnect the client from jenkins and reconnect to pick up the environment change.

**Comment by Haroon-Khel** [comment]
@sxa Could you disconnect and reconnect the machine please?

**Comment by sxa** [comment]
Done - agent relaunched on test-aws-ubuntu1804-armv8-1 :-)
